### ISSUE TITLE
Ignore Java 9's module-info classes by default

### ISSUE THREAD

**Issue by bclozel** [basepom/duplicate-finder-maven-plugin#31]
It is easy to configure this plugin to ignore duplicate `module-info.class` classes with the following:

```
<configuration>
  <ignoredClassPatterns>
    <ignoredClassPattern>module-info</ignoredClassPattern>
  </ignoredClassPatterns>
</configuration>
```

Since those classes might be quite common in the future, shouldn't this be ignored by default?
Otherwise, those are marked as duplicate classes with different content.

**Comment by whiskeysierra** [comment]
I see a similar issue now with `/META-INF/versions/9/module-info.class`

**Comment by whiskeysierra** [comment]
https://travis-ci.org/zalando/logbook/jobs/448681344

**Comment by bclozel** [comment]
@whiskeysierra Yes, see https://github.com/spring-projects/spring-boot/commit/c82a13d18e324d3dc1b7cbd37e8891f74436efa3

@hgschmie should we open a new issue or reopen this one?
